public/pages/template.html

```diff
@@ -15,24 +15,25 @@ <h1 class="cover-heading">
 <div class="input-group">
     <textarea class="form-control copy-field" id="md" readonly>
 [![GitLicense](https://gitlicense.com/badge/{{account}}/{{repo}})](https://gitlicense.com/license/{{account}}/{{repo}})</textarea>
-    <button class="btn btn-default" data-clipboard-target="#md">
-        <i class="fa fa-copy"></i>
-    </button>
+    <span class="input-group-addon btn btn-default" data-clipboard-target="#md">
+          <i class="fa fa-copy"></i>
+    </span>
 </div>
 <h2>reStructuredText</h2>
 <div class="input-group">
     <textarea class="form-control copy-field" id="re" readonly>
 .. image:: https://gitlicense.com/badge/{{account}}/{{repo}}
   :target: https://gitlicense.com/license/{{account}}/{{repo}}</textarea>
-    <button class="btn btn-default" data-clipboard-target="#re">
-        <i class="fa fa-copy"></i>
-    </button>
+    <span class="input-group-addon btn btn-default" data-clipboard-target="#re">
+          <i class="fa fa-copy"></i>
+    </span>
 </div>
 <h2>HTML</h2>
 <div class="input-group">
     <textarea class="form-control copy-field" id="ht" readonly>
 &lt;a href='https://gitlicense.com/license/{{account}}/{{repo}}'&gt;&lt;img src='https://gitlicense.com/badge/{{account}}/{{repo}}'/&gt;&lt;/a&gt;</textarea>
-    <button class="btn btn-default" data-clipboard-target="#ht">
-        <i class="fa fa-copy"></i>
-    </button>
-</div>
\ No newline at end of file
+    <span class="input-group-addon btn btn-default" data-clipboard-target="#ht">
+          <i class="fa fa-copy"></i>
+    </span>
+</div>
+</div>
```

Comment:
This line should be removed, there are no open tag for this

Reply:
Done in 2047181